docs/tutorials/ofo.md

```diff
@@ -0,0 +1,113 @@
+  - If using kind, you'll also need this cluster spec file: [kind-cluster.yaml](@site/static/samples/kind-cluster.yaml), which defines a 3-node cluster with an forwarded containerPort
+- the file defining our demo deployment, service, and CRD: [end-to-end.yaml](https://raw.githubusercontent.com/open-feature/playground/main/config/k8s/end-to-end.yaml) (more on this later).
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+- [k9s](https://k9scli.io/) (optional, if you'd like to inspect your cluster visually)
+
+### Let's see the code!
+
+<!-- TODO: add/remove this based on PR feedback -->
+ATTN REVIEWIERS: Should we include some code snippets here? or is it too much?
+
+### Show me the commands!
+
+#### Build our cluster with kind
+
+OK, let's get our cluster up and running! If you already have a K8s cluster, you can skip to [Install cert-manager](#install-cert-manager).
+
+First, create our cluster using the `kind-cluster.yaml` file:
+```shell
+kind create cluster --config kind-cluster.yaml
+```
+This might take a minute or two.
+
+#### Install cert-manager
+
+Great! Next, because our operator makes use of webhooks, we need some certificate infrastructure in our cluster.
+If your cluster already has cert manager, or you're using another solution for certificate management, you can skip to [Create Namespace](#create-namespace).
+
+Install cert-manager, and wait for it to be ready: 
+```shell
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.9.1/cert-manager.yaml && \
+kubectl wait --timeout=60s --for condition=Available=True deploy --all -n 'cert-manager'
+```
+
+#### Create namespace
+
+Next, we need to create a namespace for the operator and our workload: 
+```shell
+kubectl create namespace open-feature-operator-system
+```
+
+#### Install OpenFeature operator certificate
+
+Now let's install the Openfeature operator certificate: 
+```shell
+kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/download/v0.2.5/certificate.yaml
+```
+
+#### Install OpenFeature operator
+
+And finally, let's install the operator itself:
+```shell
+kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/download/v0.2.5/release.yaml && \
+kubectl wait --timeout=60s --for condition=Available=True deploy --all -n 'open-feature-operator-system'
+```
+
+#### Deploy our workload
+
+Now that the operator is ready to go, we can deploy our workload:
+
+```shell
+kubectl apply -f end-to-end.yaml && \
+kubectl wait --timeout=60s deployment --for condition=Available=True -l 'app=open-feature-demo'
+```
+
+Now you should see our fictional app at http://localhost:30000
```

Comment:
```suggestion
`kubectl port-forward svc/open-feature-demo-service -n default 30000:30000`
Now you should see our fictional app at http://localhost:30000
```

Reply:
This isn't necessary using the provided kind configuration which already exposes a port, but I'll make that clear.